src/Tizen.Applications.WatchfaceComplication/Tizen.Applications/EditablesContainer.cs

```diff
@@ -0,0 +1,348 @@
+            if (ret != ComplicationError.None)
+            {
+                ErrorFactory.ThrowException(ret, "Request edit fail");
+            }
+        }
+
+        /// <summary>
+        /// Overrides this method if you want to handle the behavior when the editor is ready to edit.
+        /// </summary>
+        /// <param name="editorId">The appid of ready to edit editor.</param>
+        /// <since_tizen> 6 </since_tizen>
+        protected abstract void OnEditReady(string editorId);
+
+        /// <summary>
+        /// Loads the editable's current data.
+        /// </summary>
+        /// <returns>The editable's latest data that selected by editor.</returns>
+        /// <exception cref="ArgumentException">Thrown when some parameter are invalid.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to invalid operation.</exception>
+        /// <exception cref="NotSupportedException">Thrown when the watchface complication is not supported.</exception>
+        /// <remarks>
+        /// This API loads editable's last current data which is updated by editor application.
+        /// </remarks>
+        /// <param name="editableId">The id of the editable.</param>
+        /// <example>
+        /// <code>
+        /// internal void InitEditables()
+        /// {
+        ///     _container = new MyContainer();
+        ///     Bundle curData = EditablesContainer.LoadCurrentData(_colorEditId);
+        ///     List&lt;Bundle&gt; candidatesList = new List&lt;Bundle&gt;();
+        ///     int curIdx = 0;
+        ///     int i = 0;
+        ///     foreach (string str in _colorArr)
+        ///     {
+        ///         Bundle data = new Bundle();
+        ///         data.AddItem(_colorKey, str);
+        ///         candidatesList.Add(data);
+        ///         if (curData != null &amp;&amp; curData.GetItem(_colorKey) != null
+        ///             &amp;&amp; curData.GetItem(_colorKey).Equals(str))
+        ///         {
+        ///             curIdx = i;
+        ///         }
+        ///         i++;
+        ///    }
+        ///    ColorDesign colorEdit = new ColorDesign(candidatesList, curIdx, "COLOR", _complicationBtn);
+        ///    colorEdit.Highlight = new Highlight(ShapeType.Circle, 0, 40, 10, 10);
+        ///    _container.Add(colorEdit, _colorEditId);
+        /// }
+        /// </code>
+        /// </example>
+        /// <since_tizen> 6 </since_tizen>
+        public static Bundle LoadCurrentData(int editableId)
+        {
+            SafeBundleHandle handle;
+            ComplicationError err = Interop.WatchfaceComplication.LoadCurrentData(editableId, out handle);
+            if (err == ComplicationError.None)
+                return new Bundle(handle);
+
+            return null;
+        }
+
+        /// <summary>
+        /// Releases the unmanaged resources used by the EditablesContainer class specifying whether to perform a normal dispose operation.
```

Comment:
Releases the unmanaged resources used by the EditablesContainer class specifying whether or not to perform a normal dispose operation.